bin.js

```diff
@@ -13,6 +13,12 @@ process.on('unhandledRejection', ({ message, stack, code }) => console.log({
     code
 }));
 
+process.on('exit', ({ code }) => console.log({
```

Comment:
Either use the logger, or "JSON.stringify" the message

Reply:
Also, "code" is not an attribute, but an argument
```js
process.on('exit', (code) =>
```

Reply:
I changed it